api/main_endpoints/routes/OfficeAccessCard.js

```diff
@@ -168,18 +191,33 @@ router.get('/delete', async (req, res) => {
   const tryDeleteCard = await deleteCard(cardBytes);
   if (!tryDeleteCard) {
     logger.info('Error deleting card');
-    writeRequestResponse({
+    writeLogToClient({
       statusCode: SERVER_ERROR,
       endpoint: '/delete',
       message: 'Error deleting card'
     });
     return res.sendStatus(SERVER_ERROR);
   }
   logger.info('Successfully deleted card');
-  writeRequestResponse({ statusCode: OK, endpoint: '/delete', message: 'Card deleted!' });
+  writeLogToClient({ statusCode: OK, endpoint: '/delete', message: 'Card deleted!' });
   return res.sendStatus(OK);
 });
 
+router.get('/getAllCards', async (req, res) => {
+  if (!await decodeToken(req)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  let getCards = await getAllCards();
+  if (!getCards) {
+    logger.info('Error retrieving cards');
+    return res.sendStatus(SERVER_ERROR);
+  }
+
+  logger.info('Retrieved all cards successfully!');
+  res.json(getCards).status(OK);
```

Comment:
this is a silly nitpick, the less we add `not` in our if statements the easier debugging becomes later

also we already log errors, so we can skip logging it again, plus logging nothing on success is ok
```suggestion
  const getCards = await getAllCards();
  if (getCards) {
    return res.json(getCards).status(OK);
  }
  return res.sendStatus(SERVER_ERROR);
```

Reply:
done